### PR TITLE
Removing gcsutil depedency from jacobsa/gcloud 

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Copied  gsutil package from jacobsa/gcloud repo.
Fixed lint tests.
Not copied unnecessary files [md5.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/md5.go), [crc32c.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/crc32c.go) and [doc.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/doc.go)
[jacobsa/gcloud/gcs/gcsutil](https://github.com/jacobsa/gcloud/tree/master/gcs/gcsutil)

[doc.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/doc.go) - Does not have any implementation
[md5.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/md5.go) and  [crc32c.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/crc32c.go)- Written function was not getting used anywhere in gcsfuse.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Sanity check done(Read, Write, Delete, Create operations)
2. Unit tests - NA
3. Integration tests - Integration tests are passing.

### CODE REVIEW

**Review by vadlakondaswetha — COMMENTED** [review]
can you add the details why [md5.go],(https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/md5.go), [crc32c.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/crc32c.go) and [doc.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/doc.go)

are not required. Are they not used currently in gcsfuse or did we replace it with something else.

**Comment by Tulsishah** [comment]
> can you add the details why [md5.go],(https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/md5.go), [crc32c.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/crc32c.go) and [doc.go](https://github.com/jacobsa/gcloud/blob/master/gcs/gcsutil/doc.go)
> 
> are not required. Are they not used currently in gcsfuse or did we replace it with something else.

Added

**Review by vadlakondaswetha — DISMISSED** [review]

File: internal/gcloud/gcs/gcsutil/create_objects.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	}
+
+	recordChan := make(chan record, len(input))
+	for name, contents := range input {
+		recordChan <- record{name, contents}
+	}
+
+	close(recordChan)
+
+	for i := 0; i < 10; i++ {
```
Comment (sethiay) [inline internal/gcloud/gcs/gcsutil/create_objects.go:44]: this seems like a bug on jacobsa/gcloud side. There they have created parallelism variable but not have used it. However, similar thing is done in jacobsa/gcloud/gcs/gcsutil/delete_all_objects.go. 

Can you check if parallelism=64 can be used here also?

Comment (Tulsishah) [reply internal/gcloud/gcs/gcsutil/create_objects.go]: Done

**Review by sethiay — APPROVED** [review]

**Pull request merged**